src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -95,6 +102,43 @@ public static void registerWorldGenerator(IWorldGenerator generator, int modGene
     }
 
     /**
+     * Registers a entity selector factory which is used to create predicates whenever a command containing selectors is executed
+     * Any non vanilla arguments that you expect has to be registered.
+     *
+     * @param arguments Expected string arguments in commands
```

Comment:
I think this javadoc should be made more clear for people who have never implemented an entity selector before.
In the javadoc, please give a full example command that someone might type into the console, and point out which part of it is an "expected string argument".